src/Symfony/Component/HttpFoundation/BinaryFileResponse.php

```diff
@@ -164,6 +166,10 @@ public function setContentDisposition($disposition, $filename = '', $filenameFal
         if ('' === $filenameFallback && (!preg_match('/^[\x20-\x7e]*$/', $filename) || false !== strpos($filename, '%'))) {
             $encoding = mb_detect_encoding($filename, null, true);
```

Comment:
I think we have no need to be stricter now than we were already. If people put non-utf8 string, that's going to be sent and displayed in some way by browsers, so there is already a "reporting" effect.
throwing an exception would just break code that work "fine" now. So, still on the "8bit" side :)